enterprise-suite/tests/smoke_alert

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash
+
+#set -x
+
+## Test alerting
+
+NAMESPACE=${NAMESPACE:=lightbend}
+
+source smokecommon
+source prometheus_common
+
+# 1. Create new configmap for alertmanager deployment
+# 2. Cache existing configmap setting in deployment
+# 3. Update alertmanager deployment configs to use new configmap
+# 4. run tests
+# 5. Put original configmap back in place
+
+export alertmanager_smoke_config=$(mktemp -d -t configmap.XXX)
```

Comment:
`alertmanager_smoke_config` --> `alertmanager_smoke_config_dir` would read more clearly for me. looks like things writing the same config file otherwise

Reply:
Changed.